src/components/grid/Subscriber.js

```diff
@@ -3,10 +3,15 @@
  * Subscriber pattern implementation.
  */
 class SizeSubscriber {
+  name;
   observers = [];
   width = 0;
   height = 0;
 
+  constructor(name = undefined) {
```

Comment:
This `name` prop is not needed, it was for debugging purposes to easier debug which subscribers have incremented counters.